test/unit/printer/sourcemaps.spec.ts

```diff
@@ -76,7 +76,7 @@ test.each([
     {
         code: `
             // @ts-ignore
-            class Bar extends Foo {
+            class Bar extends Foo() {
```

Comment:
Why this change?

Reply:
https://github.com/TypeScriptToLua/TypeScriptToLua/pull/774#discussion_r368480625
As a function call it gets printed on a new line, which makes it important for mapping stack traces

Reply:
I would prefer if we continue testing the most commonly used class pattern (like it was before). If you want to test this because of the new line, please just add another test.

Reply:
I'm fine with reverting it to just `Foo`, but I don't really want to add a new case just for a single mapping. So maybe just re-add `{ luaPattern: "Foo)", typeScriptPattern: "Foo {" }`? It is the thing we actually care about, even though it wouldn't be in the call stack, it's still useful for other sourcemap use cases

Reply:
sure